### PR TITLE
[feat/fix]: Update Astro and fix styling bugs

### LINKED ISSUE #220: [BUG] Style makes other pages look strange
### Problem subject

_No response_

### Description

Seems like the new website's style makes other pages look strange, such as the Community Partners form.

image

### Expected behavior/contents

The input and other parts of the website should look accordingly to the style.

### Actual behavior/contents

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Panquesito7**
#### Description of the change

- Update Astro and fix styling bugs.
  - Closes #220.

#### Screenshots

image

#### Checklist

**I agree to the following :-**

- [x] Added description of the change
- [x] I've read the [contributing guidelines](https://github.com/dev-protocol/devprotocol.xyz/blob/main/CONTRIBUTING.md)
- [x] Search previous suggestions before making a new PR, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): We should check all the pages and ensure everything is fine before merging.

### CODE REVIEW

**Comment by Panquesito7** [comment]
No idea why Vercel isn't loading Tailwind CSS. You can use Gitpod and it should look fine.

File: src/styles/base.css [hunk 1]
```diff
@@ -4,6 +4,24 @@
...
+
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #888;
+  border-radius: 10px;
+}
```
Comment (mkubdev) [inline src/styles/base.css:20]: Space for code style
```suggestion
}

```
LGTM ⚡

**Review by mkubdev — DISMISSED** [review]

**Review by vinzvinci — DISMISSED** [review]
Cool 🎉 LGTM

**Review by mkubdev — APPROVED** [review]
LGTM ⚡

**Review by KukretiShubham — APPROVED** [review]
looks good

**Review by aggre — APPROVED** [review]

**Review by vinzvinci — APPROVED** [review]
Time to merge!

**Pull request merged**